script.js

```diff
@@ -21,7 +16,7 @@ function pause() {
   document.getElementById("computers").innerHTML = "...";
 }
 
-function playGame() {
+function playGame(userChoice) {
```

Comment:
here the playGame function (which is the only use of userChoice) will accept a parameter, instead of utilizing memory from the function